tests/vector_tools/interpolate.output

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
This tests needs `MPI` and `p4est`. Hence, you should rename the output file to ` tests/vector_tools/interpolate.with_p4est=true.mpirun=1.output`